corehq/apps/reports/standard/sms.py

```diff
@@ -356,16 +356,16 @@ def headers(self):
             DataTablesColumn(_("Phone Number")),
             DataTablesColumn(_("Direction")),
             DataTablesColumn(_("Message")),
-            DataTablesColumn(_("Status")) if self.testing_changes else Ellipsis,
-            DataTablesColumn(_("Event"), sortable=False) if self.testing_changes else Ellipsis,
+            DataTablesColumn(_("Status")) if self.show_v2 else Ellipsis,
+            DataTablesColumn(_("Event"), sortable=False) if self.show_v2 else Ellipsis,
             DataTablesColumn(_("Type"), sortable=False),
         ] if header != Ellipsis])
         headers.custom_sort = [[0, 'desc']]
         return headers
 
     @cached_property
-    def testing_changes(self):
-        return toggles.SMS_LOG_CHANGES.enabled(self.request.couch_user.username)
+    def show_v2(self):
+        return toggles.SMS_LOG_CHANGES.enabled_for_request(self.request)
```

Comment:
This wasn't working properly when enabled by domain (since it'd only check username)